README.md

```diff
@@ -6,8 +6,12 @@ This repo stores SARIF Taxonomies
 
 | **Taxonomy** | **Version** | **Souce File** | **SARIF File** |
 -----|-----|-----|-----
-CWE | v4.3 | [link](https://cwe.mitre.org/data/xml/cwec/cwec_v4.3.xml.zip) | [CWE_v4.3.sarif](CWE_v4.3.sarif)
-CWE| v4.4 | [link](https://cwe.mitre.org/data/xml/cwec_v4.4.xml.zip) | [CWE_v4.4.sarif](CWE_v4.4.sarif)
+CWE | v4.3 | [link](https://cwe.mitre.org/data/xml/cwec_v4.3.xml.zip) | [CWE_v4.3.sarif](CWE_v4.3.sarif)
+CWE | v4.4 | [link](https://cwe.mitre.org/data/xml/cwec_v4.4.xml.zip) | [CWE_v4.4.sarif](CWE_v4.4.sarif)
+CWE | v4.5 | [link](https://cwe.mitre.org/data/xml/cwec_v4.5.xml.zip) | [CWE_v4.5.sarif](CWE_v4.5.sarif)
+CWE | v4.6 | [link](https://cwe.mitre.org/data/xml/cwec_v4.6.xml.zip) | [CWE_v4.6.sarif](CWE_v4.6.sarif)
+CWE | v4.7 | [link](https://cwe.mitre.org/data/xml/cwec_v4.7.xml.zip) | [CWE_v4.7.sarif](CWE_v4.7.sarif)
+CWE | v4.8 | [link](https://cwe.mitre.org/data/xml/cwec_v4.8.xml.zip) | [CWE_v4.8.sarif](CWE_v4.8.sarif)
```

Comment:
Can we get the information added to this file also reflected in the description? It would help clarify the details of the changes and allow easy reference to the source from the GitHub PR page.

Reply:
Added.